src/applications/mhv-landing-page/containers/App.jsx

```diff
@@ -78,16 +80,17 @@ const App = () => {
   useEffect(
     () => {
       async function loadMessages() {
-        const folders = await getFolderList();
-        const unreadMessages = countUnreadMessages(folders);
-        setUnreadMessageCount(unreadMessages);
+        if (hasMHVAccount) {
+          const folders = await getFolderList();
+          const unreadMessages = countUnreadMessages(folders);
+          setUnreadMessageCount(unreadMessages);
+        }
       }
-
       if (enabled) {
         loadMessages();
       }
     },
-    [enabled],
+    [enabled, hasMHVAccount],
```

Comment:
Suggestion: This could be `if (enabled && hasMHVAccount)`, and you can remove the `if` statement from `loadMessages()`.